src/components/Navigation/components/index.ts

```diff
@@ -10,11 +10,11 @@ export const StyledConnectWalletButton = styled.button`
     color: ${colors.white};
     display: flex;
     font-family: PixelSplitter;
-    font-size: .8rem;
+    font-size: 0.8rem;
```

Comment:
Are we setting `rem` somewhere?

Reply:
Yup, should be set here: https://github.com/pokt-network/wpokt-app/blob/patch/add-linting/src/index.css#L14